ens/utils.py

```diff
@@ -155,20 +178,20 @@ def raw_name_to_hash(name):
     return normal_name_to_hash(normalized_name)
 
 
-def address_in(address, addresses):
+def address_in(address: Address, addresses: Sequence[Address]) -> bool:
```

Comment:
I'm realizing most places we are currently using `Sequence` can typically use `Collection`.  We only need `Sequence` if we intend to index into data structure which is often not the case.